processing.py

```diff
@@ -0,0 +1,109 @@
+# processing Carvana image dataset
+# All input data should be kept in a folder called 'input'
+
+import skimage
+from skimage import data
+from scipy import ndimage
+import numpy as np
+import pandas as pd
+from pandas import DataFrame
+import matplotlib.pylab as plt
+import os
+from glob import glob
+
+INPUT_PATH = '../input' #Get the paths to the input data
```

Comment:
what level directory is this script going to be on?

Reply:
I just did it like this so I could save the images in the prior directory and I wouldn't have to push it on github, but we can change it.